### PR TITLE
Add continuous integration tests

### PR DESCRIPTION
**Pull request by BramBonne**
Runs tests on every push and pull request.

Additionally, this bumps the version number to trigger the tests to be picked up by AOSP's automatic importer.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Continuous integration
```
Comment (Dushistov) [inline .github/workflows/ci.yaml:1]: There is travis CI enabled for this repo: https://github.com/Nercury/android_logger-rs/blob/master/.travis.yml .
I suppose migration to github actions is ok, but we need at least convert all checks from travis to github actions.

**Review by Dushistov — CHANGES_REQUESTED** [review]
There is travis CI enabled for this repo: https://github.com/Nercury/android_logger-rs/blob/master/.travis.yml .
I suppose migration to github actions is ok, but we need at least convert all checks from travis to github actions.

**Comment by BramBonne** [comment]
Thanks for the review! I totally missed the existing Travis CI checks.

I moved everything over to GitHub actions for now, but please let me know if you prefer it when I'd just add the tests to the Travis CI instead.

**Comment by Dushistov** [comment]
Looks like without enabled github actions for repo it is impossible to test your PR,
so I created #43 . It enabled github actions for this repo when merged, and you rebase your PR,
I suppose we see results of github actions for this PR.

**Comment by BramBonne** [comment]
Sounds good!

Alternatlively, you can see my PR in action at https://github.com/BramBonne/android_logger-rs/actions as well, as it's been merged there already.

**Review by Dushistov — APPROVED** [review]
Not sure about version bump, this is depend on @Nercury . 
But otherwise looks good.

**Comment by Nercury** [comment]
This should probably only need a patch version change if the API is exactly the same. But there aren't yet many users of `0.11` version, so it's fine in this case.

**Comment by Nercury** [comment]
Let's also remove travis.

**Comment by BramBonne** [comment]
Thanks for taking a look! I changed the new version number to 0.10.1 now.

As for Travis, I removed the `.travis.yml` file in the previous patch already; is any additional action required to prevent it from triggering?

**Comment by Nercury** [comment]
I am going to sort that travis out.

**Pull request merged**